### PR TITLE
adding the new branch of checkbox

### PR DESCRIPTION
**Pull request by RazSherf**
pull request for the checkbox component

### CODE REVIEW

File: Components/Checkbox/CheckBox.module.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.main{
+    display: flex;
+    position: relative;
+
```
Comment (EkingIsrael) [inline Components/Checkbox/CheckBox.module.css:4]: @RazSherf please remove the empty lines

File: Components/Checkbox/CheckBox.module.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+}
+
+.checkBoxContainer{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+
```
Comment (EkingIsrael) [inline Components/Checkbox/CheckBox.module.css:14]: empty lines again

**Review by EkingIsrael — CHANGES_REQUESTED** [review]

File: Components/Checkbox/CheckBox.module.css (hunk 1 above)
Comment (RazSherf) [reply Components/Checkbox/CheckBox.module.css]: Fixed ✔

**Review by EkingIsrael — APPROVED** [review]

**Comment by EkingIsrael** [comment]
@RazSher@
Congratulations :) 💯 🥇

**Review by EkingIsrael — COMMENTED** [review]
great job

**Pull request closed without merging**